ShinyExpenseTracker/src/ca/ualberta/cs/shinyexpensetracker/activities/ExpenseClaimListActivity.java

```diff
@@ -126,10 +133,31 @@ public boolean onOptionsItemSelected(MenuItem item) {
 					ManageTagActivity.class);
 			startActivity(manageTagsIntent);
 			return true;
+		case R.id.set_home_geolocation:
+			Intent geolocationViewIntent = new Intent(ExpenseClaimListActivity.this,
+					GeolocationViewActivity.class);
+			startActivityForResult(geolocationViewIntent, SET_HOME_GEOLCATION);
+			return true;
 		}
 		return super.onOptionsItemSelected(item);
 	}
 
+	/*
+	 * Get result back from setting geolocation
+	 * Put it in a Toast until there is a Claimant class
+	 * where the values can actually be stored
+	 */
+	@Override
+	protected void onActivityResult(int requestCode, int resultCode, Intent data) {
+		// Check result is ok
+		if (resultCode == RESULT_OK) {
+			double latitude = data.getDoubleExtra("latitude", 39.03808);
+			double longitude = data.getDoubleExtra("longitude", 125.7296);
+			homeLocationLatitude = latitude;
+			homeLocationLongitude = longitude;
```

Comment:
Can you change these constants to the old `latitude` and `longitude` values instead of magic numbers?